prow/istio-presubmit.sh

```diff
@@ -111,6 +125,7 @@ fi
 # Build
 ${ROOT}/bin/init.sh
 
+run_or_die_on_change ./bin/fmt.sh
```

Comment:
Move this to line ~90 before any additional temporary build files are generated. Otherwise things like  `pilot/proxy/envoy/envoy` will show up in `git diff` and incorrectly fail the build.